docs/en/01_Tutorials/02_Extending_A_Basic_Site.md

```diff
@@ -462,16 +462,19 @@ Nothing here should be new. The *StaffPage* page type is more interesting though
 	:::php
 	<?php
 	class StaffPage extends Page {
-		private static $db = array(
-		);
+
 		private static $has_one = array(
-			'Photo' => 'Image'
+			'Photo' => 'File'
```

Comment:
Mentioned in person, but I don't really like that we're loosing the typing here. Maybe `File("Image")` or `File(FILETYPE_IMAGE)` or something? Will review whole diff, then comment more globally.